### ISSUE TITLE
NVML: can't get power for GPU #0, error code 999

### ISSUE THREAD

**Issue by Fluff20** [trexminer/T-Rex#358]
The miner works fine, however even with previous releases I have always had the same issue with not being able to get the power reading for my GPU as seen below.

![image](https://user-images.githubusercontent.com/64992409/113028948-c5c99d80-9183-11eb-977e-90ac8f0dedef.png)
![image](https://user-images.githubusercontent.com/64992409/113029441-599b6980-9184-11eb-9291-2d1e21393bc3.png)

Does anyone have any advice on how to resolve this?

**Comment by robertcs** [comment]
I had that problem after trying to downgrade my nvidia drivers on GTX 1660, i solved doing a clean installation of nvidia drivers. Consider to use DDU and then install the drivers u want.

**Comment by javsp** [comment]
Same here with an Nvidia 2080 Ti. I noticed this warning exacty appears each 5 minutes. I updated the latest Nvidia drivers yesterday.

**Comment by trexminer** [comment]
Duplicate of https://github.com/trexminer/T-Rex/issues/226, see the linked article in issue 226

**Comment by dimitriovic** [comment]
just wanted to share my experience with this error
`NVML: can't get power for GPU #0, error code 999`
One week ago I made some changes in my 7 gpu rig and started the rig again, and soon enough I faced for the 1st time this error.
Just in case, I went to the bios to check if everything was ok and found out that something indeed has changed. CSM support was enabled (with an asterix *), and I'm sure I did not activate it.
After disabling it, the error never showed up again.
Until today where I, again, made some changes to the rig, gpu repartition on the motherboard pcie & psu, etc.  After all finished, started the rig and again I had this error. 
remembered the bios and went to check and again I found csm support enabled again. so I disabled it and the error stopped appearing.
 I think when there is a problem to boot the system, or the system stopping abnormally, windows, maybe, try to resolve the problem and edit the bios setting like csm support.
anyway if you have this error and did not find why or how to gfet rid of it, check your bios setting, who knows maybe it will be like in my case.
good luck
